### PR TITLE
Make avatars clickable in members area

### PR DESCRIPTION
**Pull request by omeryousaf**
@Shujaat89 plz review.
I resized the default avatar, too, to have square dimensions, the previous one was messing a bit with members list layout.

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -40,4 +40,16 @@ baghiansfromtheheart.config(['$routeProvider',
...
+baghiansfromtheheart.factory('UtilityFunctions', [ '$location',
+    function( $location ) {
+        return {
+            visitMemberProfile: function ( data ) {
+                //alert(data._id);
+                $location.path('/profile/' + ( data._id || data.id ) );
+            }
+        };
+    }
 ]);
```
Comment (omeryousaf) [inline public/app.js]: done, https://github.com/omeryousaf/cbs_community/pull/19/commits/bd84ac49ae8d3d9005cb3d042c31d31338da1653.

**Comment by Shujaat89** [comment]
dost, need to show member names along with images

**Comment by omeryousaf** [comment]
done, dost! https://github.com/omeryousaf/cbs_community/pull/19/commits/1d535bbb1ef4939c47c90d75eeefce17d0b03d7e 😉

**Comment by Shujaat89** [comment]
dost make names appear alpha order

**Comment by omeryousaf** [comment]
@Shujaat89 good point, done https://github.com/omeryousaf/cbs_community/pull/19/commits/54c853f38a4cabbe39c3ebe7673e4bb54cde1ea0.

**Pull request merged**